app/assets/v2/js/grants/_new.js

```diff
@@ -277,7 +277,26 @@ Vue.mixin({
           _alert(err.message, 'danger');
         }
       });
-    }
+    },
+    onBlur(event) {
+      let vm = this;
```

Comment:
I think we can access `this` directly in the function. I don't think we need the alias.